### PR TITLE
fix(queues): display plugin name in queue log entries

### PR DESCRIPTION
**Pull request by hanshuebner**
### Summary

Log entries written by queue functions included the name of the queue, but not the name and ID of the plugin using the queue.  This would make it difficult to correlate queue log entries written by the queue consumption side to the configuration.  With this change, an explcit log_tag key can be passed to the queue when enqueuing which is shown in all log entries written by the queue.  An explicit log_tag key was chosen over a more specific 'plugin' or 'plugin_id' key as queues are also used by non-plugin functionality in EE.

### Checklist

- [X] The Pull Request has tests
- [ ] There's an entry in the CHANGELOG N/A (unreleased)
- [ ] There is a user-facing docs PR against https://github.com/Kong/docs.konghq.com - N/A (unreleased)

### Issue reference

Fixes KAG-1394

### CODE REVIEW

File: spec/03-plugins/03-http-log/01-log_spec.lua [hunk 1]
```diff
@@ -317,7 +338,7 @@ for _, strategy in helpers.each_strategy() do
 
       helpers.wait_until(function()
         local client = assert(helpers.http_client(helpers.mock_upstream_host,
-                                                  helpers.mock_upstream_port))
+          helpers.mock_upstream_port))
```
Comment (bungle) [inline spec/03-plugins/03-http-log/01-log_spec.lua:341]: This does not seem to belong to this PR.

**Review by jschmid1 — APPROVED** [review]
